db/migrate/20180924175802_add_first_name_and_last_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddFirstNameAndLastNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string
```

Comment:
You miss the required part ;)

Reply:
I will add them in another migration to avoid having to merge the existing one